### ISSUE TITLE
MSU-1 (SNES CD Audio) support

### ISSUE THREAD

**Issue by athairus** [team-phoenix/Phoenix#125]
Putting this one in before I forget again

**Comment by Druage** [comment]
Is this a libretro core?

**Comment by athairus** [comment]
No, it's a feature that is done by byuu, the programmer of bsnes.

**Comment by twinaphex** [comment]
And you can "remove" this feature again because the bsnes cores provide this "feature". This is nothing you have to "add" to a frontend and nothing you can in fact add to a frontend.

**Comment by athairus** [comment]
I want to learn more about how this works before I close it. Maybe Phoenix can make it easy to manage the extra MSU-1 data like the patch for the game or the assets themselves.
